dask/array/tests/test_array_core.py

```diff
@@ -2413,12 +2413,7 @@ def test_from_array_scalar(type_):
 
     dx = da.from_array(x, chunks=-1)
     assert_eq(np.array(x), dx)
-    assert isinstance(
-        dx.dask[
-            dx.name,
-        ],
-        np.ndarray,
-    )
+    assert isinstance(dx.dask[dx.name,], np.ndarray,)
```

Comment:
This is a whitespace change from `black`. What version does the CI use?

Reply:
It should use the latest release of `black` (today that's `20.8b1`)